tools/recognition.py

```
@@ -55,137 +79,174 @@ def signalHandler(signal, frame):
-        gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
-        rgb = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
-
-        # detect faces in the grayscale frame
-        rects = detector.detectMultiScale(
-            gray,
-            scaleFactor=1.1,
-            minNeighbors=5,
-            minSize=(30, 30),
-            flags=cv2.CASCADE_SCALE_IMAGE,
+    if Arguments.get("run_only_on_notification"):
+        if run_face_recognition == False and external_trigger == True:
```

Ruff (0.7.0)

Avoid equality comparisons to `False`; use `if not run_face_recognition:` for false checks

Replace with `not run_face_recognition`

(E712)